runtime/parachains/src/disputes/tests.rs

```diff
@@ -888,12 +686,98 @@ fn test_freeze_provided_against_supermajority_for_included() {
 	});
 }
 
+#[test]
```

Comment:
Can we also have a companion test, that unconfirmed disputes trigger an error on block import?

Reply:
Done